e2e/web/specs/play-sudoku.spec.ts

```diff
@@ -163,7 +171,7 @@ test.describe("numpad", () => {
     }) => {
       const sudokuBoard = new SudokuBoardComponent(resumeGame);
       await sudokuBoard.cell[7][7].click();
-      await sudokuBoard.note.click();
+      await sudokuBoard.page.keyboard.press("t");
       await sudokuBoard.numPad[i - 1].click();
       await sudokuBoard.cellHasNotes(7, 7, i.toString());
     });
```

Comment:
Could we have a dedicated test for all undo types like you made for pause button?

Reply:
I guess to explain more on my reasoning for this: I feel that at least 1 test should fail with the test name describing exactly what is causing the failure. 
So in this case if the toggle notes hotkeys stopped working, the test that fails is "clicking numpad should fill cell with correct note". 
But if we had dedicated tests we would have at least one test failing with the test name describing the root cause, such as "Toggle note works with all possible methods". Then the test failure will tell us exactly what is wrong.

Reply:
Thanks for the explanation, that makes a lot of sense now. I'll add it, I think it's good I switched up the way we used it in other tests too though in case we ever run into a issue that doesn't occur in isolation.